packages/calypso-products/src/translations.js

```diff
@@ -211,6 +227,10 @@ export const getJetpackProductsDescriptions = () => {
 	const backupRealtimeDescription = translate(
 		'Real-time backups save every change and one-click restores get you back online quickly.'
 	);
+	const backupT1Description = backupDailyDescription;
+	const backupT2Description = translate(
+		'Go back in time and recover all your information for up to a year, with 2TB storage space.'
```

Comment:
Can we confirm with someone on @Automattic/yamato-scan that these storage amounts are correct? `p1HpG7-cyv-p2#comment-48169` says 20GB and 1TB.

Reply:
It may actually be a good idea to parameterize the storage amount here, so we can re-use the string later; though I'm not sure how numbers with units work in that context.

Reply:
I think you want @seear or @vianasw for this confirmation.